### ISSUE TITLE
pysaml2 imports it's own deprecated module generating a warning

### ISSUE THREAD

**Issue by rectalogic** [IdentityPython/pysaml2#656]
## Code Version
4.9.0

## Expected Behavior
No DeprecationWarning

## Current Behavior

```
  /site/venv/lib/python3.7/site-packages/saml2/extension/ui.py:7: DeprecationWarning: saml2.extension.ui is deprecated; use saml2.extension.mdui instead.
    DeprecationWarning,
```
## Possible Solution

`load_extensions` should not import `saml2.extension.ui`

## Steps to Reproduce

1.
```sh-session
$ python -W default::DeprecationWarning:saml2.extension.ui -c 'from saml2 import mdstore; mdstore.load_extensions()'
/site/venv/lib/python3.7/site-packages/saml2/extension/ui.py:7: DeprecationWarning: saml2.extension.ui is deprecated; use saml2.extension.mdui instead.
  DeprecationWarning,
```

**Comment by c00kiemon5ter** [comment]
Next version will be a breaking version, as we will be removing this module.

**Comment by c00kiemon5ter** [comment]
Fixed by e29f89936c9d4ad2b0913bca3cac873f22dd63ae
